.github/workflows/cd.ethereum-contracts-stable.create-release-drafts.yml

```diff
@@ -31,16 +31,16 @@ jobs:
         id: check_versions
         run: |
           # ethereum-contracts
-          PUBLISHED_ETHEREUM_CONTRACTS_VERSION=`npm show @superfluid-finance/ethereum-contracts@latest version`
-          NEW_ETHEREUM_CONTRACTS_VERSION=`jq -r .version packages/ethereum-contracts/package.json`
+          PUBLISHED_ETHEREUM_CONTRACTS_VERSION=$(npm show @superfluid-finance/ethereum-contracts@latest version)
+          NEW_ETHEREUM_CONTRACTS_VERSION=$(jq -r .version packages/ethereum-contracts/package.json)
           if [ "$PUBLISHED_ETHEREUM_CONTRACTS_VERSION" != "$NEW_ETHEREUM_CONTRACTS_VERSION" ];then
-            echo "PUBLISH_ETHEREUM_CONTRACTS=1" >> $GITHUB_ENV
+            echo "PUBLISH_ETHEREUM_CONTRACTS=1" >> "$GITHUB_ENV"
             echo "ethereum-contracts changed: $PUBLISHED_ETHEREUM_CONTRACTS_VERSION -> $NEW_ETHEREUM_CONTRACTS_VERSION"
           else
             echo ethereum-contracts unchanged: $PUBLISHED_ETHEREUM_CONTRACTS_VERSION
           fi
 
-          echo "::set-output name=NEW_ETHEREUM_CONTRACTS_VERSION::$NEW_ETHEREUM_CONTRACTS_VERSION"
+          echo "name=NEW_ETHEREUM_CONTRACTS_VERSION" >> $NEW_ETHEREUM_CONTRACTS_VERSION
```

Comment:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/